projects/ng-dynamic-component/src/lib/dynamic.component.ts

```diff
@@ -41,6 +45,10 @@ export class DynamicComponent<C = unknown>
   ndcDynamicProviders?: StaticProvider[] | null;
   @Input()
   ndcDynamicContent?: Node[][];
+  @Input()
+  ndcDynamicNgModuleRef?: NgModuleRef<any>;
+  @Input()
+  ndcDynamicEnvironmentInjector?: EnvironmentInjector;
```

Comment:
```suggestion
  ndcDynamicNgModuleRef?: NgModuleRef<unknown>;
  @Input()
  ndcDynamicEnvironmentInjector?: EnvironmentInjector | NgModuleRef<unknown>;
```

Reply:
Option "Allow edits from maintainers" is checked. Also I added you as a collaborator to my fork.